_ml-commons-plugin/connectors.md

```diff
@@ -35,35 +74,21 @@ PUT /_cluster/settings
     "persistent": {
         "plugins.ml_commons.trusted_connector_endpoints_regex": [
             "^https://runtime\\.sagemaker\\..*\\.amazonaws\\.com/.*$",
-            "^https://api\\.openai\\.com/.*$",
-            "^https://api\\.cohere\\.ai/.*$",
-            "^https://bedrock\\..*\\.amazonaws.com/.*$"
+            "^https://api\\.openai\\.com/.*$"
         ]
     }
 }
 ```
 {% include copy-curl.html %}
 
-### Enabling ML nodes
-
-Most connectors require the use of dedicated ML nodes. To make sure you have ML nodes enabled, update the following cluster settings:
-
-```json
-PUT /_cluster/settings
-{
-    "persistent": {
-        "plugins.ml_commons.only_run_on_ml_node": true,
-    }
-}
-```
-{% include copy-curl.html %}
-
-If you are running a remote inference or local model, you can set `"plugins.ml_commons.only_run_on_ml_node"` to `false` and use data nodes instead.
 
 
 ### Setting up connector access control
 
-To enable access control on the connector API, use the following cluster setting:
+If you plan on using a remote connector, making sure to use an OpenSearch cluster with the Security plugin enabled. 
```

Comment:
Add more explanation ? https://github.com/opensearch-project/documentation-website/pull/4554#discussion_r1272651401
> We should add some security warning for remote connector. As user need to configure credential in connector, for example AWS credential, openAI API key, user should always use security enabled cluster. Otherwise their credentials will not be protected which is risky.